css/style.css

```diff
@@ -679,67 +674,82 @@ footer {
 /* navigation */
 
 #navbar {
-  padding-left: 5%;
-  max-width: 1000px;
-  position: relative;
-  display: flex;
-  justify-content: flex-end;
-  align-items: flex-start;
-  top: 15px;
-  box-sizing: content-box;
   line-height: 30px;
+  display: flex;
+  align-items: center;
 }
 
 #navbar a {
   color: #666;
-  padding-left: 20px;
-  -webkit-font-smoothing: subpixel-antialiased;
 }
 
 #navbar a.active {
   font-weight: bold;
   color: #353535;
 }
 
+#navbar ul {
+  list-style: none;
+}
+
+#navmenu {
+  display: flex;
+  gap: 20px;
+}
+
 
 /* dropdown menu */
 
-#navmenu {
-  padding: 0;
-  margin: 0;
-  right: 0px;
-  z-index: 1000;
-  margin-right: 15px;
+.submenu {
+  position: relative;
+  list-style: none;
 }
 
-#navmenu > li {
+.submenu-content {
+  position: absolute;
+  top: 100%;
+  left: 50%;
+  transform: translate(-50%);
+  margin: auto;
+  box-shadow: 0px 1px 3px #00000026;
+  z-index: 1000;
+  min-width: 230px;
+  width: fit-content;
+  background-color: #fcfcfa;
+  border: 1px solid #b2b2b2;
+  border-radius: 12px;
   list-style: none;
   padding: 0;
-  float: left;
+  display: none;
 }
 
-#navmenu a:hover {
-  text-decoration: none;
+.submenu-content li:first-child a {
+  border-radius: 12px 12px 0 0;
 }
 
-.menu ul {
-  display: none;
+.submenu-content li:last-child a {
+  border-radius: 0px 0px 12px 12px;
 }
 
-.menu ul.dropit-submenu {
-  background-color: #fcfcfa;
-  border: 1px solid #b2b2b2;
-  margin: 0 0 0 1px;
-  box-shadow: 0px 1px 3px rgba(0,0,0,0.15);
+.submenu.open .submenu-content {
+  display: initial;
 }
 
-.menu ul.dropit-submenu a {
+#navmenu {
+  padding: 0;
+  margin: 0;
+  right: 0px;
+  z-index: 1000;
+  margin-right: 15px;
+}
+
+.submenu-content li a {
   display: block;
   color: #777;
-  padding: 0 8px 2px 8px;
+  padding: 2px 20px 2px 20px;
 }
 
-.menu ul.dropit-submenu a:hover {
+.submenu-content a:hover {
   background: #259dff;
   color: #fff !important;

```

Comment:
This is the important that causes problems. Maybe this can go? 
I suggest we remove this and never use important (see dark-theme.css for more)